<?xml version='1.0' encoding='UTF-8'?>
<BugCollection version='4.5.3' threshold='medium' effort='default'><file classname='io.extact.rms.client.api.dto.ReservationClientDto'><BugInstance type='EI_EXPOSE_REP' priority='Normal' category='MALICIOUS_CODE' message='io.extact.rms.client.api.dto.ReservationClientDto.getRentalItemDto() は，ReservationClientDto.rentalItemDto を返すことによって内部表現を暴露するかもしれません。' lineNumber='25'/><BugInstance type='EI_EXPOSE_REP' priority='Normal' category='MALICIOUS_CODE' message='io.extact.rms.client.api.dto.ReservationClientDto.getUserAccountDto() は，ReservationClientDto.userAccountDto を返すことによって内部表現を暴露するかもしれません。' lineNumber='26'/><BugInstance type='EI_EXPOSE_REP2' priority='Normal' category='MALICIOUS_CODE' message='io.extact.rms.client.api.dto.ReservationClientDto.setRentalItemDto(RentalItemClientDto) は，ReservationClientDto.rentalItemDto の中に外部の可変オブジェクトを格納することによって内部表現を暴露するかもしれません。' lineNumber='14'/><BugInstance type='EI_EXPOSE_REP2' priority='Normal' category='MALICIOUS_CODE' message='io.extact.rms.client.api.dto.ReservationClientDto.setUserAccountDto(UserAccountClientDto) は，ReservationClientDto.userAccountDto の中に外部の可変オブジェクトを格納することによって内部表現を暴露するかもしれません。' lineNumber='14'/></file><file classname='io.extact.rms.client.api.exception.ValidateClientException'><BugInstance type='EI_EXPOSE_REP2' priority='Normal' category='MALICIOUS_CODE' message='new io.extact.rms.client.api.exception.ValidateClientException(ValidateClientException$ValidationErrorMessage) は，ValidateClientException.errorMessage の中に外部の可変オブジェクトを格納することによって内部表現を暴露するかもしれません。' lineNumber='12'/></file><file classname='io.extact.rms.client.api.exception.ValidateClientException$ValidationErrorMessage'><BugInstance type='EI_EXPOSE_REP' priority='Normal' category='MALICIOUS_CODE' message='io.extact.rms.client.api.exception.ValidateClientException$ValidationErrorMessage.getErrorItems() は，ValidateClientException$ValidationErrorMessage.errorItems を返すことによって内部表現を暴露するかもしれません。' lineNumber='41'/><BugInstance type='EI_EXPOSE_REP2' priority='Normal' category='MALICIOUS_CODE' message='io.extact.rms.client.api.exception.ValidateClientException$ValidationErrorMessage.setErrorItems(List) は，ValidateClientException$ValidationErrorMessage.errorItems の中に外部の可変オブジェクトを格納することによって内部表現を暴露するかもしれません。' lineNumber='44'/></file><file classname='io.extact.rms.client.api.login.LoggedInEvent'><BugInstance type='EI_EXPOSE_REP' priority='Normal' category='MALICIOUS_CODE' message='io.extact.rms.client.api.login.LoggedInEvent.getLoginUser() は，LoggedInEvent.loginUser を返すことによって内部表現を暴露するかもしれません。' lineNumber='15'/><BugInstance type='EI_EXPOSE_REP2' priority='Normal' category='MALICIOUS_CODE' message='new io.extact.rms.client.api.login.LoggedInEvent(UserAccountClientDto) は，LoggedInEvent.loginUser の中に外部の可変オブジェクトを格納することによって内部表現を暴露するかもしれません。' lineNumber='12'/></file><Error></Error><Project><SrcDir>C:\reps\rms\rms-client-api\src\main\java</SrcDir><SrcDir>C:\reps\rms\rms-client-api\src\test\java</SrcDir></Project></BugCollection>